pom.xml

```diff
@@ -234,7 +234,23 @@
         <dependency>
             <groupId>software.amazon.awssdk.iotdevicesdk</groupId>
             <artifactId>aws-iot-device-sdk</artifactId>
-            <version>1.20.1</version>
+            <!--
+            When updating the version here, ensure you match the correct aws-crt version below.
+            Get the correct version from: https://github.com/aws/aws-iot-device-sdk-java-v2/blob/main/sdk/pom.xml#L45
+            !-->
+            <version>1.20.2</version>
```

Comment:
out of curiosity, was there an issue with 1.20.1?

Reply:
Maybe, but I don't think so. Just taking the latest that they just released.